test/unit/Execution/DeliveryExecutionStatusManagerTest.php

```diff
@@ -90,7 +90,7 @@ public function testGetExecutionsInProgressReturnsArrayOfObjects()
 
         $result = $this->object->getExecutionsInProgress();
 
-        $this->assertInternalType('array', $result, 'Method must return array.');
+        $this->assertisarray($result, 'Method must return array.');
```

Comment:
are you sure that this should not be camel case ?

Reply:
Nice catch. And this was in a lot of places. Thanks!